core/src/main/java/com/linecorp/armeria/server/Route.java

```diff
@@ -61,7 +61,35 @@ static RouteBuilder builder() {
      * @see RouteBuilder#matchesHeaders(Iterable)
      * @see RouteBuilder#matchesParams(Iterable)
      */
-    RoutingResult apply(RoutingContext routingCtx);
+    default RoutingResult apply(RoutingContext routingCtx) {
```

Comment:
Better deprecating and removing all references to this method?

Reply:
That's a good suggestion. Fixed. 😉